bitnami/nginx/1.27/debian-12/rootfs/opt/bitnami/scripts/libnginx.sh

```diff
@@ -208,6 +208,10 @@ nginx_initialize() {
     fi
     nginx_configure "absolute_redirect" "$(is_boolean_yes "$NGINX_ENABLE_ABSOLUTE_REDIRECT" && echo "on" || echo "off" )"
     nginx_configure "port_in_redirect" "$(is_boolean_yes "$NGINX_ENABLE_PORT_IN_REDIRECT" && echo "on" || echo "off" )"
+
+    if [[ "${NGINX_ENABLE_STREAM}" != "no" ]]; then
```

Comment:
You can use this instead:
```suggestion
    if is_boolean_yes "$NGINX_ENABLE_STREAM"; then
```

Reply:
ok, I changed it